index.html

```diff
@@ -35,7 +37,7 @@ <h2 class="hero__table__header">Oakland Election Live Snapshot</h2>
             <thead class="hero__table__head">
                 <tr>
                     <th></th>
-                    <th class="hero__column__header">MARCH 2020</th>
+                    <th class="hero__column__header">{{ primary_date | date: "%B %Y" | upcase }}</th>
                     <th class="hero__column__header">NOVEMBER 2020</th>
```

Comment:
This should use election_date

Reply:
In your image in the pull request it shows NOVEMBER 2022, so maybe you did not commit all your changes?

Reply:
Why is this still NOVEMBER 2020?

Reply:
ok I see this  is changed in a later commit.